src/stories/disclosure/Disclosure.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { Disclosure, DisclosureProps } from "./Disclosure";
+
+export default {
+  title: "Atoms / Disclosure",
```

Comment:
Please rename this to adapt to the Library structure.

Reply:
👍 Right!